src/platforms/affinity/index.ts

```diff
@@ -0,0 +1,17 @@
+import { client } from '@/platforms/affinity/client';
+import { icon } from '@/platforms/affinity/icon';
+import { auth, platform } from '@/sdk';
+
+export default platform('affinity', {
+  auth: auth.basic({
+    questions: [{ type: 'text', id: 'password', label: 'API Token' }],
```

Comment:
Don't we need both a username and a password question for basic auth? I'm looking at the basic auth PR, it looks like it needs an answer at https://github.com/vesselapi/integrations/pull/157/files#diff-0883450c0459616d32193a78474a5358d92fcc1785ad4820ecba2d23792b926cR130

Reply:
so the funny thing is that some of them don't require `username` or `password`

Reply:
Hence the `??`

Reply:
ehhhhhh say what? Like the basic auth is literally `base64('password:')` where password is the api token?

Reply:
Yup, I've seen both 
```
base64(`${API token}:`}

and 

base64(`:${API token}`}
```